### PR TITLE
pages/1.11/security: distinguish between EE and OSS security subsections

### PR DESCRIPTION
**Pull request by gpaul**
## Description
This PR aligns the Security subsections' titles in `/pages/1.11` to that of `/pages/1.10`.

https://jira.mesosphere.com/browse/DCOS-20752

## Urgency
- [ ] Blocker
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by adamtheturtle** [comment]
Uhhh I can't build the docs:

```
  metalsmith-revision 4776 files restored +10m
  metalsmith-timer CB: Reduce +5ms
/Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:76
          callback(null);
          ^

RangeError: Maximum call stack size exceeded
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:76:11
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/gaze.js:442:5
    at iterate (/Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:69:5)
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:78:11
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/gaze.js:442:5
    at iterate (/Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:69:5)
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:78:11
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/gaze.js:442:5
    at iterate (/Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:69:5)
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:78:11
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/gaze.js:442:5
    at iterate (/Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:69:5)
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:78:11
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/gaze.js:442:5
    at iterate (/Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:69:5)
    at /Users/Adam/Documents/mesosphere/mesosphere/dcos-docs-site/node_modules/gaze/lib/helper.js:78:11
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mesosphere-docs@0.0.1 dev: `NODE_ENV=development DEBUG=metalsmith-timer,metalsmith-revision,metalsmith-algolia node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mesosphere-docs@0.0.1 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Adam/.npm/_logs/2018-02-05T21_54_17_465Z-debug.log
make: *** [Makefile:30: build-development] Error 1
```

(that's on staging)

This looks good but please can you post before and after screenshots, then I should approve.

**Comment by jongiddy** [comment]
I have tested it and confirm that it solves the issue. Only query (possibly for the docs folks) is whether only the `navigationTitle` should be changed, and the page title left as Security? This is triggered by the appearance of the Enterprise logo after the page title. But I'm happy either way.

**Comment by gpaul** [comment]
@adamtheturtle I had to hack some random js files to get this to work. I'm looking forward to a fully dockerized local dev process for docs (not quite volunteering, yet).

**Comment by gpaul** [comment]
@jongiddy 

> Only query (possibly for the docs folks) is whether only the navigationTitle should be changed, and the page title left as Security?

I decided to make this change to align 1.11 with what we had in 1.10. Maybe that's the wrong way around :-) @pavisandhu PTAL?

File: pages/1.11/security/ent/index.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 layout: layout.pug
-navigationTitle:  Security
-title: Security
+title: DC/OS Enterprise Security
```
Comment (pavisandhu) [inline pages/1.11/security/ent/index.md]: Please set the metadata as follows (for consistency with DC/OS Open Source Security section. 

navigationTitle: DC/OS Enterprise Security
title: DC/OS Enterprise Security

The former controls the text in the navigation panel, and the latter controls the title of the page.

Comment (gpaul) [reply pages/1.11/security/ent/index.md]: Thanks for the explanation @pavisandhu, PTAL.

Comment (pavisandhu) [reply pages/1.11/security/ent/index.md]: If everything looks as expected in your local build, please merge.

**Pull request merged**